### ISSUE TITLE
image names are wrong in the csv file

### ISSUE THREAD

**Issue by bugger69** [wangzhifengharrison/HTNet#6]
The image names in the combined_3_class2_for_optical_flow.csv are wrong and hence the file is impossible to run..

**Comment by wangzhifengharrison** [comment]
What do you mean for the wrong image names? All the images are provided in the '/datasets'. I already provded the codes to deal with image names. such as 'sub01_EP02_01f.jpg' to '
sub01_EP02_01f .png'. If you have any questions, feel free to ask. Thanks.

**Comment by wangzhifengharrison** [comment]
I can upload the weights of pre-trained model to google drive so you can download from google drive if you prefer. Or you just run the train code, the weights will be saved in the file.

**Comment by bugger69** [comment]
Yeah man that would be great if that's convenient for you, cuz training is kind of  taking forever. Thanks.
